app/models/manageiq/providers/autosde/inventory/parser/storage_manager.rb

```diff
@@ -181,11 +181,35 @@ def storage_service_resource_attachments
     end
   end
 
-  # This method changes capability name field from 'abstract_capability' to 'name'
-  def parse_capabilities(capabilities, field_name)
+  def parse_ems_capabilities(capabilities)
+    return capabilities unless capabilities.is_a?(Array)
+
+    caps_hash = {}
+    capabilities.each do |capability|
+      name = capability.abstract_capability
+      (caps_hash[name] ||= []) << {"uuid" => capability.uuid, "value" => capability.value}
+    end
+    caps_hash
```

Comment:
```suggestion
    capabilities.to_h do |capability|
      [capability.abstract_capability, {"uuid" => capability.uuid, "value" => capability.value}]
    end
```

Reply:
Just noticed you expect to have >1 value per key.  You can use `each_with_object` instead and you can use `Hash.new { |k, v| k[v] = []}` so you don't need the `(caps_hash[name] ||= [])`

Try `capabilities.each_with_object(Hash.new { |k, v| k[v] = []}) do |capability, result|`